sources/packages/backend/libs/services/src/queue/queue.service.ts

```diff
@@ -37,4 +54,49 @@ export class QueueService {
       dashboardReadonly: queue.queueConfiguration.dashboardReadonly,
     }));
   }
+
+  /**
+   * Get queue configuration for the requested queue name.
+   * @param queueName queue name
+   * @returns queue configuration.
+   */
+  async getQueueConfiguration(queueName: QueueNames): Promise<Bull.JobOptions> {
+    const queueConfig = await this.queueConfigurationDetails(queueName);
+    const config = {} as Bull.JobOptions;
+    const queueConfiguration = queueConfig.queueConfiguration;
+    if (queueConfiguration.retry && queueConfiguration.retryInterval) {
+      config.attempts = queueConfiguration.retry;
```

Comment:
Question for this PR or not, why we did not keep the same name for the JSONB object?
For instance, why did we "renamed" `attempts` to `retry` and `backoff` to `retryInterval`?

Reply:
oh.... after was renamed after the discussion with @dheepak-aot as `retry ` and `retryInterval` for meaningful and in the future if we are planning to go for any queue U, thought these names are helpful.

Reply:
IMO, I do not see a point here. The `retryInterval` is actually the `backoff ` that assumes different configurations.

![image](https://user-images.githubusercontent.com/61259237/207724858-3af15c73-1b78-457f-8c1a-f08c0853fafb.png)

![image](https://user-images.githubusercontent.com/61259237/207725022-ca727a17-365b-4c6b-8473-97aa6fcb9a72.png)

![image](https://user-images.githubusercontent.com/61259237/207725168-5e22484e-4744-497a-9323-4236c5ae9185.png)

Reply:
Yes you are right. But to use the custom types, we need to define the custom strategy function. which needs to be used in the code and only retry interval if that is set as backoff could be something that can come from config. 

![image](https://user-images.githubusercontent.com/54600590/207728501-be8dd73f-a818-4c03-8340-51e9ef3dd47d.png)